### PR TITLE
Fix a problem where frost-select would clear prompt on redraw

### PR DESCRIPTION
**Pull request by TheAxeMan301**
# PATCH

Fix a problem where frost-select would clear prompt on redraw

### CODE REVIEW

**Comment by vesper2000** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-core/pull-request/193/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-core)

File: addon/components/frost-select.js [hunk 1]
```diff
@@ -240,15 +240,18 @@ export default Component.extend(PropTypeMixin, {
     .fromPairs()
     .value()
 
-    this.get('reduxStore').dispatch(resetDropDown(reduxAttrs))
-
     const dataChanged = isAttrDifferent(newAttrs, oldAttrs, 'data')
     const selectedChanged = isAttrDifferent(newAttrs, oldAttrs, 'selected')
     const selectedValueChanged = isAttrDifferent(newAttrs, oldAttrs, 'selectedValue')
 
+    if (!(selectedChanged || selectedValueChanged)) {
```
Comment (sandersky) [inline addon/components/frost-select.js:247]: I find that `if (!selectedChanged && !selectedValueChanged) {` reads a bit better as it doesn't have the extra parenthesis that the negation applies to.

**Comment by sglanzer-fire** [comment]
Makes sense, ideally we should get the test working - can we put another few hours of effort into that (maybe consult @juwara0) and if we can't get it in that time range we'll fall back to the manual test for now?

**Comment by TheAxeMan301** [comment]
Added a test that does fail without my fix. Thanks @juwara0 !

**Comment by sglanzer-fire** [comment]
Approved - thanks 👍 

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-core/pull-request/193/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-core)

**Pull request merged**